pallets/migrations/src/lib.rs

```diff
@@ -305,7 +323,7 @@ pub mod pallet {
 				<MigrationState<T>>::insert(migration_name_as_bytes, true);
 
 				weight = weight.saturating_add(consumed_weight);
-				if weight > available_weight {
+				if weight.ref_time() > available_weight.ref_time() {
```

Comment:
It will be great when we can also check against PoV here :)

Reply:
Yeah, didnt want to include for now...I guess it can be a follow-up PR